gammapy/spectrum/models.py

```diff
@@ -1716,3 +1717,53 @@ def energy_flux(self, emin, emax):
         return a * (np.exp(-u_min ** 2) - np.exp(-u_max ** 2)) + b * (
             erf(u_max) - erf(u_min)
         )
+
+
+class SpectralLogGaussian(SpectralModel):
+    r"""Gaussian spectral model.
+
+    .. math::
+
+        \phi(E) = \frac{N_0}{E \, \sigma \sqrt{2\pi}}  \exp{ \frac{\left( \ln(\frac{E}{\bar{E}}) \right)^2 }{2 \sigma^2} }
+
+    This model was used in this CTA study for the electron spectrum: Table 3 in https://ui.adsabs.harvard.edu/abs/2013APh....43..171B
+
+
+    Parameters
+    ----------
+    norm : `~astropy.units.Quantity`
+        :math:`N_0`
+    mean : `~astropy.units.Quantity`
+        :math:`\bar{E}`
+    sigma : `float`
+        :math:`\sigma`
+
+
+    Examples
+    --------
+    This is how to plot the default `Gaussian` spectral model:
```

Comment:
Remove "default". There's two Gaussian spectral models now, one is not more default than the other.

Reply:
You could reference the other from the docstring and explain with 1 sentence how they differ, or overplot them here in this example to show that they are very similar, but not exactly the same shape.